CleanWeather/Scenes/CityHistorical/CityHistoricalWorker.swift

```diff
@@ -23,9 +25,28 @@ final class CityHistoricalWorkerImpl {
 
 extension CityHistoricalWorkerImpl: CityHistoricalWorker {
 
-    func fetchCityHistoricalWeather(id: String, date: Date, completion: FetchHistoricalCompletion?) {
+    func fetchCityHistoricalWeather(cityDetails: CityWeather, date: Date, completion: FetchHistoricalCompletion?) {
         let convertedDate = unixFormatDate(date: date)
```

Comment:
to date formatter

Reply:
That private func unixFormatDate has nothing to do with DateFormatter.
.timeIntervalSince1970 is native Apple Date extension. Is it worth to make my own function in DateFormatterHeleper for that?
I can do it even simpler. Remove all private func unixFormatDate and pass date.timeIntervalSince1970 to networking.fetchHistoricalWeatherForCity(...)
`networking.fetchHistoricalWeatherForCity(coordinates: coordinates.coordinatesString(), date: date.timeIntervalSince1970) `

Or maybe Im missing something?

Reply:
id youd like to display it then it would tequile you to convert the value to a date. You could have a separate method for that or split it somehow in current methods